config/prow/cluster/starter.yaml

```diff
@@ -3,24 +3,149 @@
 # their own files.
```

Comment:
perhaps add to line 2/3 comment.  Something like:
"For examples see the per component yaml files in this directory."